package.json

```diff
@@ -1,25 +1,34 @@
 {
-  "name": "blueprint-monorepo",
+  "name": "blueprintjs-monorepo",
   "version": "1.33.0",
   "private": true,
   "description": "A React UI toolkit for the web.",
   "workspaces": [
     "packages/*"
   ],
   "scripts": {
-    "bootstrap": "lerna bootstrap",
-    "build:landing": "(cd packages/site-landing; npm run build)",
-    "build:gulp": "NODE_ENV='\"production\"' gulp build",
-    "build:site": "npm-run-all build:gulp build:landing clean:docs copy:docs copy:landing delete:doc-source-maps",
-    "clean:docs": "rm -rf docs && git checkout docs/CNAME",
-    "copy:docs": "cp -rf packages/site-docs/dist docs/docs",
-    "copy:landing": "cp -rf packages/site-landing/dist/* docs/.",
-    "delete:doc-source-maps": "find docs -name '*.map' -type f -delete",
-    "gulp": "gulp",
+    "bundle-apps": "lerna run bundle",
+    "compile-libs": "lerna run compile",
+    "clean": "lerna run clean",
+    "clean-gh-pages": "rm -rf docs && git checkout docs/CNAME",
+    "copy-docs-app": "cp -rf packages/docs-app/dist docs/docs",
+    "copy-landing-app": "cp -rf packages/landing-app/dist/* docs/.",
+    "dev:all": "lerna run dev",
+    "dev:core": "lerna run dev --scope '@blueprintjs/{core,docs-app}'",
+    "dev:docs": "lerna run dev --scope '@blueprintjs/docs-app'",
+    "dev:datetime": "lerna run dev --scope '@blueprintjs/{core,datetime,docs-app}'",
+    "dev:labs": "lerna run dev --scope '@blueprintjs/{core,labs,docs-app}'",
+    "dev:table": "lerna run dev --scope '@blueprintjs/table'",
```

Comment:
Should this be `'@blueprintjs/{core,table}'`, since `table` uses utilities from `core`?